@here/harp-mapview/lib/Tile.ts

```diff
@@ -1413,13 +1420,23 @@ export class Tile implements CachedResource {
                     }
                 }
 
+                const extrudedObjects: Array<{
+                    object: THREE.Object3D;
+                    materialFeature: boolean;
+                }> = [];
+
                 const renderDepthPrePass =
                     technique.name === "extruded-polygon" && isRenderDepthPrePassEnabled(technique);
 
                 if (renderDepthPrePass) {
                     const depthPassMesh = createDepthPrePassMesh(object as THREE.Mesh);
                     objects.push(depthPassMesh);
 
+                    extrudedObjects.push({
```

Comment:
The depthPassMesh and the edgeObj are always added to the extrudedObjects whereas the object is added only if:
`                    isExtrudedPolygonTechnique(technique) &&
                    animatedExtrusionHandler !== undefined`

Reply:
You are right. It shouldn't be like that. I'll correct it